src/App.js

```diff
@@ -1,7 +1,25 @@
+import Header from "./Components/Header";
+import NavBar from "./Components/NavBar";
+import Articles from "./Components/Articles";
+import { Route, Routes } from "react-router-dom";
 import "./App.css";
 
 function App() {
-  return <div className="App"></div>;
+  return (
+    <div className="App">
+      <Header>
+        <Header />
+      </Header>
+      <nav>
```

Comment:
Similar to above, move these nav tags into your NavBar and return them from there